docs/the-new-architecture/pure-cxx-modules.md

```diff
@@ -0,0 +1,472 @@
+# Pure C++ Turbo Native Modules
+
+import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem'; import constants from '@site/core/TabsConstants';
+
+Writing a module in C++ is the best way to share platform-agnostic code between Android and iOS. With pure C++ modules, you can write your logic only once and reuse it right away from all the platform, without the need of writing platform specific code.
+
+In this guide, we will go through the creation of a pure C++ Turbo Native Module:
+
+1. Create the JS specs
+2. Configure Codegen to generate the scaffolding
+3. Implement the Native logic
+4. Register the module in the Android and iOS application
+5. Test your changes in JS
+
+The rest of this guide assume that you have created your application running the command:
+
+```shell
+npx @react-native-community/cli@latest init SampleApp --version 0.76.0
+```
+
+## 1. Create the JS specs
+
+Pure C++ Turbo Native Modules are Turbo Native Modules. They needs a specification file (also called spec file) so that Codegen can create the scaffolding code for us. The Specification file is also what we use to access the Turbo Native Module in JS.
```

Comment:
```suggestion
Pure C++ Turbo Native Modules are Turbo Native Modules. They needs a specification file (also called spec file) so that Codegen can create the scaffolding code for us. The specification file is also what we use to access the Turbo Native Module in JS.
```